src/actions/object.js

```diff
@@ -1,6 +1,16 @@
 import axios from 'axios';
+import bodybuilder from 'bodybuilder';
 import * as ActionTypes from '../constants';
 
+// todo: refactor to de-duplicate this logic from the ./objects.js file
```

Comment:
I didn't want to refactor this yet cause I think you were playing with this stuff still. Feel free to refactor it or I can later. (or it wouldn't be the worst thing to leave it).